doc/train/finetuning.md

```
@@ -250,13 +247,12 @@ one can select a specific branch (e.g., `CHOOSEN_BRANCH`) included in `multitask
-:::{note}
-One can check the available model branches in multi-task pre-trained model by refering to the documentation of the pre-trained model or by using the following command:
-
-```bash
-$ dp --pd show multitask_pretrained.pd model-branch
-```
-:::
+> [!NOTE]
+> One can check the available model branches in multi-task pre-trained model by refering to the documentation of the pre-trained model or by using the following command:
+>
+> ```bash
+> $ dp --pd show multitask_pretrained.pd model-branch
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)